lib/vdc/NetCDFCFCollection.cpp

```diff
@@ -14,6 +14,17 @@ using namespace VAPoR;
 using namespace Wasp;
 using namespace std;
 
+namespace {
+vector<string> make_unique(vector<string> v)
+{
+    sort(v.begin(), v.end());
```

Comment:
STD has a function named `make_unique` so I don't think it's a good idea to make your custom function the same name.

Reply:
done